src/Ztm.WebApi/TransactionConfirmationWatchers/TransactionConfirmationWatcher.cs

```diff
@@ -2,29 +2,28 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using NBitcoin;
 using Ztm.Threading;
+using Ztm.WebApi.Callbacks;
 using Ztm.Zcoin.Synchronization;
 using Ztm.Zcoin.Watching;
 
-using Rule = Ztm.WebApi.TransactionConfirmationWatchingRule<Ztm.WebApi.TransactionConfirmationCallbackResult>;
 using Timer = Ztm.Threading.Timer;
 
-namespace Ztm.WebApi
+namespace Ztm.WebApi.TransactionConfirmationWatchers
```

Comment:
I think `TransactionConfirmationWatchers` is too long. My idea is split it into `Watchers.TransactionConfirmation`.